app/views/privacy_policies/show.html.erb

```diff
@@ -0,0 +1,103 @@
+    <p class="govuk-body">Manage Training for Early Career Teachers is a new service.  We’re confident that it has met our security standards, but we’ll continue to run security tests as we improve the service.</p>
+
+  <h2 class="govuk-heading-m">What personal data we collect</h2>
+    <p class="govuk-body">The personal data we collect will depend on whether you’re an early career teacher, mentor, induction tutor/coordinator or a training provider.</p>
+    <h3 class="govuk-heading-s">What data we collect if you're an early career teacher</h3>
+      <p class="govuk-body">If you’re an early career teacher using Manage Training For Early Career Teachers we may collect the following information about you:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>name</li>
+          <li>address</li>
+          <li>date of birth</li>
+          <li>phone number</li>
+          <li>email address</li>
+          <li>TRN number</li>
+          <li>any other information you choose to share durin the applicaiton process - for example, you can choose to share equality and diversity information</li>
+        </ul>
+    <h3 class="govuk-heading-s">What data we collect if you work for a training provider</h3>
+      <p class="govuk-body">If you process teacher training applications, we may need to collect information about you so we can process applications, for example:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>your name</li>
+          <li>your phone number</li>
+          <li>your email address</li>
+          <li>where you work</li>
+          <li>what your role is</li>
+        </ul>
+
+  <h2 class="govuk-heading-m">How we use your data</h2>
+    <h3 class="govuk-heading-s">Processing applications</h3>
+      <p class="govuk-body">Processing applications includes the following:</p>
+        <ul class="govuk-list govuk-list--bullet">
+          <li>Sending applications to training providers</li>
+          <li>managing communications between candidates and training providers</li>
+          <li>working out and funding candidates are entitled to</li>
+          <li>getting in touch if there is a security issue around your data</li>
+          <li>getting in touch with candidates and training providers about their applications</li>
+        </ul>
+    <h3 class="govuk-heading-s">Building a better teacher training application process</h3>
+      <p class="govuk-body">For example, we'll look at any feedback you give us about our services so we can improve them.</p>
+      <p class="govuk-body">If you're an early career teacher or you work at a teacher training provider, we may contact you to carry out user research.</p>
+    <h3 class="govuk-heading-s">Getting insight to make government policy better</h3>
+      <p class="govuk-body">We might analyse your data to help us inform government policy around teacher recruitment and retention.</p>
```

Comment:
I prefer indentation to match html structure, rather than document structure. Can we change this to conform to that?